src/features/game/lib/factions.ts

```diff
@@ -260,331 +259,285 @@ type MonthlyFactionPrize = {
   items?: Partial<Record<InventoryItemName, number>>;
 };
 
-export const BONUS_FACTION_PRIZES: Record<
-  string,
-  Record<number, MonthlyFactionPrize>
-> = {
-  "2024-07-22": {
-    1: {
-      wearables: {
-        "Crimstone Hammer": 1,
+export const FACTION_BONUS_WEEKS = [
+  "2024-10-28",
```

Comment:
?

Reply:
Good eye - this was just used in some Backend tests as a dummy week